test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -246,6 +246,7 @@
  * <li> - a random seed used to initialize the index random context.
  * </ul>
  */
+@LuceneTestCase.SuppressCodecs("*") // remove this once we have asserting codec that works with ES
```

Comment:
So this stops wrapping sorted doc values with `AssertingSortedDocValues`?

Reply:
yes